modules/ROOT/pages/architecture/hexagonale_architecture.adoc

```diff
@@ -194,6 +194,169 @@ This makes the application more difficult to understand and harder to locate the
 
 Therefore, we propose to implement the domain model as anemic entities and make usage of use cases and services to implement the business logic and interact with the domain models.
 
+== Module-based hexagonal component architecture
+The previously described approach of structuring using package structures is particularly suitable for lightweight and smaller applications.
+
+For larger applications, it makes sense to additionally implement the division using Maven or Gradle modules.
+This has the advantage that the application core of the hexagon remains completely without special technical frameworks such as JPA.
```

Comment:
From my point of view, I would structure the article a bit differently. For me this sentences of the technoogy framework free core comes a bit out of nowhere. 
I would suggest the following:

1. Start with the business component division. Why have we choosen it (mainly for a future proof structure when more components arise during time)
2. Then the division into core and adapter. Here answering why diving them again (For me here the technology free core belongs) 
3. The bootstrap module at the end. Explaining that it's necessary to define it here, so that the starting point of the applicaiton has the change to collect all dependencies necessary for the DI framework are collected. Hence it needs dependencies to all modules available.